vm/validate.go

```diff
@@ -83,13 +87,15 @@ func verifyNullObject(t *testing.T, i int, obj Object) bool {
 
 func verifyStringObject(t *testing.T, i int, obj Object, expected string) bool {
 	t.Helper()
+	var fuzStr string
 	switch result := obj.(type) {
 	case *StringObject:
-		if result.value != expected {
+		re, _ := regexp2.Compile("(?<=#<[a-zA-Z0-9_]+:)[0-9]{12}(?=[ ]>)", 0)
+		fuzStr, _ = re.Replace(result.value, "XXXXXXXXXXXX", 0, -1)
```

Comment:
So we need to use a fixed number of "X"s? It'd be hard to remember how may Xs we need to put. How about a flag-like value, like `[ANY_VALUE]`? And in the future, we can have several flags like `[INTEGERS]` to help the specs (this one is not necessary, just make an example).

Reply:
I got it! I feel "XXX..." looks a bit clumsy😅 and I'd revise that.